plutus-core/src/Language/PlutusCore/Evaluation/Machine/Cek.hs

```diff
@@ -379,9 +379,8 @@ computeCek ctx env (Builtin ex bn) = do
   arity <- arityOf bn
   returnCek ctx (VBuiltin ex bn arity arity [] [] env)
 -- s ; ρ ▻ error A  ↦  <> A
-computeCek _ _ (Error _ ty) =
-    throwingWithCause _EvaluationError (UserEvaluationError CekEvaluationFailure) $ Just err where
-        err = Error () $ void ty
+computeCek _ _ Error{} =
+    throwingWithCause _EvaluationError (UserEvaluationError CekEvaluationFailure) $ Nothing
```

Comment:
I unfortunately had to remove this in order to get the CEK machine to print the same message as the CK machine when it encountered an error so that I could run the golden tests on the CK machine as well.  We can't make the CK machine error message include the error term itself because (I think) it wants a `CkValue` instead of a `Term`.  The error message used to say something like `caused by `(error a_0)`, which wasn't too informative anyway; it's still a bit annoying to lose it though.

Reply:
>  it wants a CkValue instead of a Term.

I think that's because I stupidly forgot to update it the way I updated the CEK machine. Feel free to make it `Nothing` for now, I'll update the CK machine and we'll see if we can make it `Just` back.